### PR TITLE
Add recipe for varnet 1.5.3

### PR DESCRIPTION
**Pull request by kiranchari**
Describe your pull request here

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

### General instructions

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

### Instructions for avoiding API, ABI, and CLI breakage issues
Conda is able to record and lock (a.k.a. pin) dependency versions used at build time of other recipes.
This way, one can avoid that expectations of a downstream recipe with regards to API, ABI, or CLI are violated by later changes in the recipe.
If not already present in the meta.yaml, make sure to specify `run_exports` (see [here](https://bioconda.github.io/contributor/linting.html#missing-run-exports) for the rationale and comprehensive explanation).
Add a `run_exports` section like this:

```yaml
build:
  run_exports:
    - ...

```

with `...` being one of:

| Case                             | run_exports statement                                               |
| -------------------------------- | ------------------------------------------------------------------- |
| semantic versioning              | `{{ pin_subpackage("myrecipe", max_pin="x") }}`     |
| semantic versioning (0.x.x)      | `{{ pin_subpackage("myrecipe", max_pin="x.x") }}`   |
| known breakage in minor versions | `{{ pin_subpackage("myrecipe", max_pin="x.x") }}` (in such a case, please add a note that shortly mentions your evidence for that) |
| known breakage in patch versions | `{{ pin_subpackage("myrecipe", max_pin="x.x.x") }}` (in such a case, please add a note that shortly mentions your evidence for that) |
| calendar versioning              | `{{ pin_subpackage("myrecipe", max_pin=None) }}`    |

while replacing `"myrecipe"` with either `name` if a `name|lower` variable is defined in your recipe or with the lowercase name of the package in quotes.

### Bot commands for PR management

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

Note that the <code>@BiocondaBot please merge</code> command is now depreciated. Please just squash and merge instead.

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by kiranchari** [comment]
@thanhleviet can this PR be merged? Thank you

File: recipes/varnet/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  sha256: b97c5acf60ad16d5061b23ad34f6d5210bbf6667393872fdd56ee1876eb554ba
+  patches:
+    # Stray `if` line in the v1.5.3 tag; fixed upstream on master after tagging.
+    - fix-indels-predict-syntax.patch
+
+build:
+  number: 0
+  noarch: python
+  run_exports:
+    - {{ pin_subpackage(name, max_pin="x.x") }}
```
Comment (thanhleviet) [inline recipes/varnet/meta.yaml]: `run_exports` uses `max_pin="x.x"`, but per Bioconda convention `max_pin="x.x"` is meant for pre-1.0 (`0.x.x`) releases; for semantic versioning at `1.x+` (this recipe is `1.5.3`) the convention is `max_pin="x"` unless there's known evidence of breakage within minor versions. If there's a specific reason to lock the minor version here, a short note would help; otherwise consider switching to `max_pin="x"`.

File: recipes/varnet/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    - pip
+  run:
+    - python >={{ python_min }}
+    - numpy
+    - pandas >=1.1.1
+    - joblib >=0.16.0
+    - pysam >=0.16.0.1
+    # Bundled models were saved with Keras 2; TF >=2.16 switches tf.keras to Keras 3,
+    # which cannot load them.
+    - tensorflow-cpu >=2.13,<2.16
```
Comment (thanhleviet) [inline recipes/varnet/meta.yaml:36]: Upstream `requirements.txt` pins `tensorflow==2.3.0` exactly, and this recipe relaxes to `tensorflow-cpu >=2.13,<2.16` (a large jump, justified in the commit message by the Python/TF version-solving constraint and the Keras 2/3 model-loading issue). That reasoning makes sense, but the `test:` section only runs `--help`, so the actual model-loading/inference path across this much larger TF range isn't exercised by CI. Could you confirm you've run an actual prediction (not just `--help`) against the bundled models with a TF version in this range, to make sure loading works end-to-end?

File: recipes/varnet/build.sh [hunk 3]
```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+TARGET=$PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM
+mkdir -p $TARGET
+cp -r * $TARGET/
+
+mkdir -p $PREFIX/bin
+
+# Wrapper for predict.py
```
Comment (thanhleviet) [inline recipes/varnet/build.sh]: This comment just narrates the heredoc immediately below it.

```suggestion
```

File: recipes/varnet/build.sh [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+cat << EOF > $PREFIX/bin/varnet-predict
+#!/bin/bash
+python $TARGET/predict.py "\$@"
+EOF
+chmod +x $PREFIX/bin/varnet-predict
+
+# Wrapper for filter.py
```
Comment (thanhleviet) [inline recipes/varnet/build.sh]: Same here — narrates the next line.

```suggestion
```

**Review by thanhleviet — COMMENTED** [review]
Thanks for the recipe. CI passes, sha256 checks out, license/license_file look correct, and the build.sh approach (copy tree + wrapper scripts) is appropriate since upstream has no setup.py. A couple of points before this is ready to merge, plus two small nits.

**Comment by kiranchari** [comment]
@thanhleviet Thanks for the careful review — the end-to-end request was well placed, it found two real bugs.

Fixed the two comment nits and switched run_exports to max_pin="x".

On the TF range: I ran a full prediction (not just --help) on a synthetic tumor/normal pair with one somatic SNV and one 3bp deletion. Both bundled models load and score under tensorflow-cpu 2.15: SNV chr1:10001 T>A SCORE=0.9997, deletion chr1:12000 TAAT>T SCORE=0.9974, both PASS in the VCF.

That run exposed two issues the --help tests miss:

- pandas 3 makes np.split/np.array_split on a DataFrame return ndarrays, so prediction batching fails with IndexError in snvs/predict.py (also on upstream master). Capped pandas <3.
- bedtools was missing from run deps. predict.py calls bedtools sort via os.system for the final VCF; absent the binary it fails silently in an && chain and the VCF is left unsorted. Added bedtools plus a bedtools --version test.

**Comment by kiranchari** [comment]
@thanhleviet just following up on this pull request. Kindly approve if all checks pass. Thanks!

**Review by thanhleviet — APPROVED** [review]
All feedback addressed: comment nits fixed, run_exports switched to max_pin="x", and the TF-range concern was verified with an actual end-to-end prediction (not just --help), which also surfaced and fixed the pandas 3 / missing bedtools issues. CI is green. Thanks for the thorough follow-up.

**Comment by thanhleviet** [comment]
@BiocondaBot please add label
